.github/workflows/ci.yml

```diff
@@ -29,10 +29,34 @@ jobs:
           apk add --no-progress --update git
           mix deps.get
       - run: mix test
+      # - run: MIX_ENV=docs mix inch.report
```

Comment:
I wasn't sure what to do with this, so I left it with the original grouping (`test-elixir`)

Reply:
Yup, leave it here, remove the one from the installer. :)